app/config/platforms.php

```diff
@@ -139,6 +139,22 @@
         'enabled' => true,
         'beta' => false,
         'languages' => [ // TODO change key to 'sdks'
+            [
+                'key' => 'cli',
+                'name' => 'Command Line',
+                'version' => '0.1.0',
+                'url' => 'https://github.com/appwrite/sdk-for-cli',
+                'package' => 'https://github.com/appwrite/sdk-for-cli',
```

Comment:
For package don't we need to point to the docker image ?

Reply:
It's just used for a link on the docs page 👍🏻